gemma-core/src/main/java/ubic/gemma/core/genome/gene/service/GeneServiceImpl.java

```diff
@@ -123,12 +123,14 @@ public Gene findByEnsemblId( String exactString ) {
     @Override
     @Transactional(readOnly = true)
     public Gene findByNCBIId( Integer accession ) {
+        if ( accession == null ) return null;
```

Comment:
We should traceback where those nulls came from. If it's the frontend, it should be treated as a `400 Bad Request`.

Reply:
See https://github.com/PavlidisLab/Gemma/issues/1008